src/artwork.c

```diff
@@ -507,34 +499,23 @@ artwork_get(struct evbuffer *evbuf, char *path, struct evbuffer *inbuf, int max_
   return ART_E_ERROR;
 }
 
-/* Looks for an artwork file in a directory. Will rescale if needed.
- *
- * @out evbuf     Image data
- * @in  dir       Directory to search
- * @in  max_w     Requested width
- * @in  max_h     Requested height
- * @out out_path  Path to the artwork file if found, must be a char[PATH_MAX] buffer
- * @return        ART_FMT_* on success, ART_E_NONE on nothing found, ART_E_ERROR on error
- */
 static int
-artwork_get_dir_image(struct evbuffer *evbuf, char *dir, int max_w, int max_h, char *out_path)
+find_dir_image(const char *dir, char *out_path)
```

Comment:
Extra style thing: I suggest we keep the verbs last, so dir_image_find(), keep the output arguments first, and don't make any assumptions about the output buffer. So it would become dir_image_find(char *out_path, size_t len, const char *dir).